CHANGELOG.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 
 master
 ------
+- (`#116 <https://github.com/GranthamImperial/silicone/pull/116>`_) Updated to work with new version of pyam
```

Comment:
```suggestion
- (`#116 <https://github.com/GranthamImperial/silicone/pull/116>`_) Updated to work with pyam v0.7
```

v0.8 is now out, do we want to go up to that?

Reply:
So this code does now go up to 0.8 (it was still a type of 0.7 when I wrote it).